src/app/pages/send/ordinal-inscription/send-inscription-form.tsx

```diff
@@ -37,7 +39,17 @@ export function SendInscriptionForm() {
       onSubmit={chooseTransactionFee}
     >
       <Form>
-        <BaseDrawer title="Send" enableGoBack isShowing onClose={() => navigate(RouteUrls.Home)}>
+        <Dialog
```

Comment:
@kyranjamie @fbwoolf : This change has stopped the `Send Ordinal` flow from moving to the `Choose Fee` step. 

I'm not sure whats happening. I tried:
- moving the submit button back where it was
- using Formik props and calling `props.handleSubmit`
- moving the form inside the dialog

Its like the Formik hooks can't run inside of the dialog. Maybe I need to use `forwardRef` or something like that? 

I faced a similar issue in the `Receive` modal where the `use-copy-to-clipboard` was not firing correctly. 

There's also a newly introduced issue since Fara implemented the new `Toast` where the toasts don't fire in the dialog. 

I can keep checking it but maybe there's something obvious I'm missing? 

I also need to prioritise [this task](https://github.com/leather-wallet/extension/issues/4467) adding tests to the flow. 

There seems to be some kind of form validation in the choose fee screen but I didn't see any validation fire for empty address etc.

Reply:
@pete-watters I made a couple of commits here. The toasts are working in the Dialogs, but in the receive modal there weren't any toasts. After I added them, they do show. However, it appears the overlay of the Dialog is on top of it? And, in the popup and the smaller viewports they do appear under the Dialog. That will likely have to be fixed with how the toast is positioned in the `Toast.Viewport`. I didn't get that far yet but can help if needed.

![Screenshot 2024-03-14 at 7 45 59 PM](https://github.com/leather-wallet/extension/assets/6493321/bbda1f87-7d57-4c3e-a6df-cff3839f3044)

I also did get the send inscription form advancing to the fees screen as long as I did pass the props.handleSubmit() to the footer button, that might be bc it is passed to another component? I'm not sure.

<img width="532" alt="Screenshot 2024-03-14 at 7 45 34 PM" src="https://github.com/leather-wallet/extension/assets/6493321/61028119-25dc-4b9a-901c-1592c3935418">

Reply:
Made edits to my last comments, got the toasts working ^.

Reply:
That's brilliant, thanks @fbwoolf ! 👏  I had tried using `props.handleSubmit()` but didn't get it working. It could have been another error I was making or not rebuilding or something. Anyway, who cares if you got it working now! 

I made another change this morning and force pushed before seeing your commits so I will add them in again now and squash everything